content/_help/verify-your-identity/verify-your-identity-in-person/issues-with-verifying-your-identity-in-person._en.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Issues with verifying your identity in person
+meta_title: Issues with verifying your identity in person
+child: false
+order: 2
+permalink: /help/verify-your-identity/verify-your-identity-in-person/issues-with-verifying-your-identity-in-person/
+---
+## I received an email from Login.gov, but I did not attempt to verify my identity at a Post Office
+
+Sign into your Login.gov account and [change your password immediately](https://login.gov/help/manage-your-account/change-your-password/). If you’d like, you can also [delete your account](https://login.gov/help/manage-your-account/delete-your-account/).
```

Comment:
Can you change these URLs (and others in this pull request) to be root-relative?

1. The canonical URL is intended to be `www.` ([ref](https://github.com/GSA-TTS/identity-site/blob/b3619209eac6dca37b0445fd6dee7aa6d4da25bc/_config.yml#L4))
2. So that URLs in local development and preview sites point to the article in that same environment

```suggestion
Sign into your Login.gov account and [change your password immediately](/help/manage-your-account/change-your-password/). If you’d like, you can also [delete your account](/help/manage-your-account/delete-your-account/).
```

Reply:
Done. See commit f45bc5d1284a7d8021a78da2ceaaa7aad704e7cb.
Searched for more instances of `](https://login.gov` and made more updates with commit c7afb47e5cde0602f56a9b9d9dc5f7d94650339a. These needed language selector in url as well.